src/vector/platform/WebPlatform.ts

```diff
@@ -46,6 +46,14 @@ export default class WebPlatform extends VectorBasePlatform {
         return 'Web Platform'; // no translation required: only used for analytics
     }
 
+    /**
+     * Return true if platform supports multi-language
+     * spell-checking, otherwise false.
+     */
+    supportsMultiLanguageSpellCheck(): boolean {
```

Comment:
I believe this duplicates `BasePlatform` without changes, so I don't think we need to repeat it here.